### PR TITLE
feat: add mail3 interaction

### PR DESCRIPTION
**Pull request by ghcoldsnap**
Since we don't have much resources to implement this feature either, the dropdown component uses a third-party library.

Sorry for the inconvenience.

Ref:
https://www.figma.com/file/XrV9eeaR2c19ElpyvDk7Pe/bit.host?node-id=0%3A1

### CODE REVIEW

File: src/pages/-c/ProfileCard.vue [hunk 1]
```diff
@@ -154,16 +210,23 @@ import { defineComponent, onMounted, ref } from '@nuxtjs/composition-api'
...
+import 'floating-vue/dist/style.css'
 
 export default defineComponent({
   name: 'ProfileCard',
   components: {
     DasAvatar,
     Iconfont,
+    IconProfile,
+    Dropdown,
+    IconProfile,
```
Comment (web3max) [inline src/pages/-c/ProfileCard.vue:225]: IconProfile repeats the declaration.

File: src/pages/-c/ProfileCard.vue [hunk 2]
```diff
@@ -123,15 +157,37 @@
...
-
-      <button class="profile_share_button _card" type="button" @click="toggleCard">
-        <Iconfont name="card" />
-        {{ $tt('Share Card') }}
-      </button>
+      <Dropdown>
+        <button class="profile_button">
+          <IconProfile name="share" size="16" />
+          <span class="profile_button_text">{{ $tt('Share') }}</span>
+          <IconProfile name="dropdown-black" :size="10" />
```
Comment (web3max) [inline src/pages/-c/ProfileCard.vue:164]: the size is of type String.

File: src/pages/-c/ProfileCard.vue [hunk 3]
```diff
@@ -123,15 +157,37 @@
...
+              <span>{{ $tt('Share Card') }}</span>
+            </button>
+          </div>
+        </template>
+      </Dropdown>
+      <Dropdown>
+        <button class="profile_button profile_button_contact">
+          <IconProfile name="contact" size="16" />
+          <span class="profile_button_text">{{ $tt('Contact') }}</span>
+          <IconProfile name="dropdown-white" :size="10" />
```
Comment (web3max) [inline src/pages/-c/ProfileCard.vue:183]: the size is of type String.

File: src/pages/-c/ProfileCard.vue [hunk 4]
```diff
@@ -123,15 +157,37 @@
...
+        </button>
+        <template #popper>
+          <div class="popover_content">
+            <component :is="mail3MeButton" v-if="mail3MeButton" />
+          </div>
```
Comment (web3max) [inline src/pages/-c/ProfileCard.vue:188]: The content needs to be centered on the left.

**Comment by ghcoldsnap** [comment]
@web3max 
All mentioned issues have been fixed.
Also, the contact icon will displays a red dot when there are unread messages. Since the dropdown component `floating-vue` will load the poper content only when it is displayed for the first time, a line of workaround code has been added. I'm not familiar with Vue, so if you have a better way, feel free to suggest it.

**Pull request merged**